simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/FOSS5xEmptyRepo.scala

```diff
@@ -0,0 +1,71 @@
+package com.puppetlabs.gatling.node_simulations
+import com.puppetlabs.gatling.runner.SimulationWithScenario
+import org.joda.time.LocalDateTime
+import org.joda.time.format.ISODateTimeFormat
+import java.util.UUID
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+// import io.gatling.jdbc.Predef._
+
+class FOSS5xEmptyRepo extends SimulationWithScenario {
+
+// 	val httpProtocol = http
+// 		.baseURL("https://sbbr2c2w66jmcpm.delivery.puppetlabs.net:8140")
+// 		.acceptHeader("application/json, text/pson")
+// 		.acceptEncodingHeader("gzip;q=1.0,deflate;q=0.6,identity;q=0.3")
+// 		.userAgentHeader("Puppet/5.3.3 Ruby/2.4.2-p198 (x86_64-linux)")
```

Comment:
Can we get rid of this comment?

Reply:
Probably. I think the script doesn't edit destructively. A lot of the other scala files have similar comments.

Reply:
Yeah, this looks like the input to what creates baseHeaders below. I don't think it should be needed? /cc @haus

Reply:
agree.

Reply:
Think it might be good to delete all of the commented lines in here?